src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,41 +1,70 @@
+import React, { useEffect } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { getUser } from '../../api';
+import { useState } from 'react';
+import { User } from '../../types/User';
+import { currentTodoSlice } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const [currentUser, setCurrentUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    if (currentTodo) {
+      getUser(currentTodo.userId)
```

Comment:
Ensure that `getUser` handles errors appropriately. If it doesn't, consider adding a `.catch()` block to handle potential errors when fetching user data.